README.md

```diff
@@ -30,73 +30,74 @@
 
 ### Motivation
 
-In January
-2020 [GitHub announced a shortcut to compare across two releases](https://github.blog/changelog/2020-01-13-shortcut-to-compare-across-two-releases/)
-. This shortcut basically leaves you
-to [pick a release version to compare with from another release version working as base](https://help.github.com/en/github/administering-a-repository/comparing-releases)
-. Then you'll see a diff of the code between those two versions. This is a nice addition, so you can see at once all
-code changes between a range of versions. However, this is not what I expected at all or what I would find most useful
-when comparing releases.
-
-What did I expect then? I spend a lot of time comparing releases, either for upgrading dependencies at work or in my
-personal projects. Or even just to be up to date about some library updates. This process is tedious, specially if there
-are a lot of releases between base and target versions as you need to paginate between multiple pages. I usually do this
-process over GitHub releases tab or going through the `CHANGELOG.md` if available. Usually I'm interested in looking for
-breaking changes more than any other kind of changes.
+In January 2020 [GitHub announced a shortcut to compare across two releases](https://github.blog/changelog/2020-01-13-shortcut-to-compare-across-two-releases/).
+This shortcut basically leaves you to [pick a release version to compare with from another release version working as base](https://help.github.com/en/github/administering-a-repository/comparing-releases).
```

Comment:
There was some odd whitespace just after the Markdown link. Which is now fixed. 😄